TLV/src/ringbuffer.c

```diff
@@ -13,7 +13,7 @@
  * @param buffer it is a structure type to access the correct ring buffer.
  * @param element it is the character.
  */
-bool put(RingBuffer* buffer, char element)
+bool put(RingBuffer* buffer, uint16_t element)
 {
 	//Check space
 	 if (hasSpace(&buffer) != 0)
```

Comment:
On line 21-23, for example, check for tabbing. You can enable a "Show Whitespace" type option in Eclipse to ensure that all tabbing is done by the tab character instead of the space character.

Be sure to do that to ensure consistent tabbing if, for example, a text editor decides to use a different number of characters per tab.